build.sbt

```diff
@@ -36,35 +36,20 @@ lazy val gen = crossProject(JSPlatform, JVMPlatform)
   .in(file("gen"))
   .dependsOn(core)
   .settings(
-    defaultSettings_3 ++ Seq(
+    defaultSettings ++ Seq(
       name    := "Graph Gen",
       version := Version.gen
     )
   )
  */
 
-lazy val coreTestScala3 = project
```

Comment:
Sorry that the intention is not commented, but this submodule is about proving that **2.13 core** can be used from Scala 3 including Scala 3 syntax like [WastelandSpec](https://github.com/scala-graph/scala-graph/blob/2.x/coreTestScala3/src/test/scala/WastelandSpec.scala) already does. Firther, it is planed to rewrite all other tests to Scala 3 syntax. This would not be possible in a cross-compiled project where we need to stick with Scala 2.13 syntax, so let's keep it.